src/app/login/page.js

```diff
@@ -1,11 +1,108 @@
 "use client";
+import React, { useState } from "react";
+import { Form, Button, Alert } from "react-bootstrap";
+
+const Login = () => {
+  const [inputUsername, setInputUsername] = useState("");
+  const [inputPassword, setInputPassword] = useState("");
+
+  const [show, setShow] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setLoading(true);
+    await delay(500);
+    console.log(`Username :${inputUsername}, Password :${inputPassword}`);
+    if (inputUsername !== "admin" || inputPassword !== "admin") {
+      setShow(true);
+    }
+    setLoading(false);
+  };
+
+  const handlePassword = () => {};
+
+  function delay(ms) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+  }
 
-const page = () => {
   return (
-    <div>
-    <h2>This will be the login page.</h2>
+    <div
+      className="sign-in__wrapper"
+      style={{ backgroundImage: `url(${BackgroundImage})` }}
+    >
+      {/* Overlay */}
+      <div className="sign-in__backdrop"></div>
+      {/* Form */}
+      <Form className="shadow p-4 bg-white rounded" onSubmit={handleSubmit}>
+        {/* Header */}
+        <img
+          className="img-thumbnail mx-auto d-block mb-2"
+          src={Logo}
```

Comment:
Logo is not defined, this throws an error when trying to see the form. If its for later use, you can just comment it out for now.